applications/salk-portal/geppetto_local_setup.sh

```diff
@@ -0,0 +1,19 @@
+yarn global add yalc
```

Comment:
why do we need a local geppetto?

Reply:
We don't need. It is a good to have when the fixes for the problems we have in salk need to happen on the geppetto side.